pages/admin/login.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import jwt_decode from "jwt-decode";
+const LitJsSdk = require('lit-js-sdk');
+import { Spinner } from '@/common/components/elements/Spinner';
+
+const accessControlConditions = [
+  {
+    method: "balanceOfToken",
+    params: ["6855BL6aaQW9NaJUtbMqyYXnewjWdnSrQxdarKQ54fdD"],
+    chain: "solana",
+    returnValueTest: {
+      key: "$.amount",
+      comparator: ">",
+      value: "0",
+    },
+  },
+];
+
+const resourceId = {
+    baseUrl: process.env.NEXT_PUBLIC_BASE_URL,
+    path: process.env.NEXT_PUBLIC_ADMIN_PATH,
+    orgId: "none",
+    role: "none",
+    extraData: "none"
```

Comment:
Looks like the recommended values are empty strings according to [the docs](https://lit-protocol.github.io/lit-js-sdk/api_docs_html/#resourceid).
```suggestion
    orgId: "",
    role: "",
    extraData: ""
```